Clients/GUI/wwwroot/css/site.css

```diff
@@ -183,7 +132,7 @@ app {
 }
 
 .newsbar {
```

Comment:
Can you move custom styles to another css file?

Reply:
fixed